src/utils/SecretManagerService.ts

```diff
@@ -81,6 +81,16 @@ export class SecretManagerService implements ISecretManagerService {
     return certs
   }
 
+  async deleteSecretAtPath (path: string): Promise<void> {
+    // to permanently delete the key, we use metadata path
+    const basePath = Environment.Config.secrets_path.replace('/data/', '/metadata/')
```

Comment:
@Walt-Intel can you add the following test to secretManagerService.test.ts:
`test('should remove device from secret store', async () => {
  const deleteSecretAtPathSpy = jest.spyOn(secretManagerService, 'deleteSecretAtPath').mockResolvedValue(null)
  const result = await secretManagerService.deleteSecretAtPath('test')
  expect(deleteSecretAtPathSpy).toHaveBeenCalledWith('test')
  expect(result).toBe(null)
})`